autometa/common/external/samtools.py

```diff
@@ -42,49 +42,46 @@ def sort(sam, bam, nproc=mp.cpu_count()):
         </path/to/alignment.sam>
     bam : str
         </path/to/output/alignment.bam>
-    nproc : int, optional
+    cpus : int, optional
         Number of processors to be used. By default uses all the processors of the system
 
     Raises
     ------
     TypeError
-        nproc must be an integer greater than zero
+        cpus must be an integer greater than zero
     FileNotFoundError
         Specified path is incorrect or the file is empty
-    ChildProcessError
-        Samtools did not run successfully, returns the return code from subprocessing call
+    subprocess.CalledProcessError
+        Samtools did not run successfully
```

Comment:
This is now raising an `ExternalToolError`

Reply:
This needs to be updated to `ExternalToolError`

```suggestion
    ExternalToolError
        Samtools did not run successfully, returns subprocess traceback and command run
```